apps/src/sites/studio/pages/levels-lablab-main.js

```diff
@@ -0,0 +1,60 @@
+import $ from 'jquery';
```

Comment:
I like having a lablab entrypoint 🎉

Reply:
+1! one thing that comes to mind is handling locale files - maybe the entrypoint just loads all the locale files for lab-lab compatible apps? don't know how large that would be though.

Reply:
For now that's exactly what is happening, in `_lab2.html.haml`.